main/mariadb/APKBUILD

```diff
@@ -28,49 +29,13 @@ source="https://downloads.mariadb.org/interstitial/mariadb-$pkgver/source/mariad
 	$pkgname.initd
 	fix-mysql-install-db-path.patch
 	fix-ucontext-check.patch
-	ppc-remove-glibc-dep.patch
 	pcre.cmake.patch
+	my-medium.cnf
 	"
-# dbug test fails under rootbld
-#options="!check"
-builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
Please stop removing `builddir` from aports.

Reply:
Same as the comment above.

Reply:
I'm quite sure that maintainer of this aport did not tell you to remove `builddir=`. I have no idea why you and one or two other contributors have started removing `builddir=`.